build.cmd

```diff
@@ -485,7 +493,15 @@ if /i "%__DoCrossArchBuild%"=="1" (
     set __MsbuildWrn=/flp1:WarningsOnly;LogFile=!__BuildWrn!
     set __MsbuildErr=/flp2:ErrorsOnly;LogFile=!__BuildErr!
 
-    @call %__ProjectDir%\run.cmd build -Project=%__CrossCompIntermediatesDir%\install.vcxproj -configuration=%__BuildType% -platform=%__CrossArch% -MsBuildLog=!__MsbuildLog! -MsBuildWrn=!__MsbuildWrn! -MsBuildErr=!__MsbuildErr! %__RunArgs% %__UnprocessedBuildArgs%
+    if not defined NumberOfEnabledCore (
+    REM Determine number of physical processor cores available on machine
+      for /f "tokens=*" %%I in (
+        'wmic cpu get NumberOfEnabledCore /value ^| find "=" 2^>NULL'
+      ) do set %%I
+    )
+    echo Number of available CPU cores %NumberOfEnabledCore%
```

Comment:
Physical cores... I assume you picked this over logical cores by measurements?

Reply:
Yes, this result of optimization runs, however to some extent it was expected as compilation is processor bound and most of HPC loads run best when hyperthreading is off and are optimized with regard to number of physical cores.